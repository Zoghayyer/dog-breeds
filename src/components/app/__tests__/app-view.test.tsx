import * as React from 'react';
import { shallow } from 'enzyme';
import AppView from '../app-view';

describe('(View) AppView', () => {
  let component;
  const defaultProps: any = {
    history: {},
  };

  const shallowAppView = (extendedProps = {}) => {
    component = shallow(<AppView {...defaultProps} {...extendedProps} />);
  };

  beforeEach(() => shallowAppView());

  it('renders without crashing', () => {
    shallow(<AppView {...defaultProps} />);
  });

  it('it renders the view component', () => {
    expect(component).toMatchSnapshot();
  });
});
