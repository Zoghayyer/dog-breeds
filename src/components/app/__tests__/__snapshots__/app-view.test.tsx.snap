// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(View) AppView it renders the view component 1`] = `
<Router
  history={Object {}}
>
  <CoreView />
</Router>
`;
